src/fusor/models.py

```diff
@@ -4,20 +4,18 @@
 from enum import Enum
 from typing import Any, Literal
 
-from ga4gh.vrsatile.pydantic import return_value
-from ga4gh.vrsatile.pydantic.vrsatile_models import (
-    CURIE,
-    GeneDescriptor,
-    LocationDescriptor,
-    SequenceDescriptor,
+from ga4gh.core.domain_models import Gene
+from ga4gh.vrs.models import (
+    LiteralSequenceExpression,
+    SequenceLocation,
 )
+from gene.schemas import CURIE
```

Comment:
Is there some other place where CURIE is defined and usable? Is it okay that I'm importing it from the gene normalizer?

Reply:
that's a good question. probably fine to pull it from the gene normalizer but that's sort of a weird place to be getting it from. could also redefine it here, either way would be fine IMO

Reply:
while some of these fields will have to stay as CURIEs, I think there were some (assay?) that we wanted to change to simple W3 URIs?

Reply:
can i include that in this PR since I'm already basically doing a big overhaul? or should I open a separate PR for this?

Reply:
@katiestahl If it's an easy change, I say include it here. Otherwise, separate PR

Reply:
going to address this in a different pr